activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -29,6 +29,11 @@ def unstub_all!
         @stubs = {}
       end
 
+      def stubbed?(object, method_name)
+        key = [object.object_id, method_name]
```

Comment:
SimpleStubs is suddenly creating arrays in 3 places, would suggest a refactor to a Hash:

```ruby
@stubs = Hash.new { |h,k| h[k] = {} }

# Which would be written to like:
@stubs[Time.object_id][:now] = Time.now
```

This would then make this method:

```ruby
def stubbed?(object, method_name)
  @stubs[object.object_id][method_name]
end
```